### DISCUSSION TITLE
Content history bug with field "select multiple"

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#47141] [2. Cold Features]
### Steps to reproduce the issue

Create filed in own/custom component  (type - list, multiple)

```
<field name="mylistvalue" type="list" default="" required="true" multiple="true" label="Select an option" description="">
  <option value="">Please Select</option>
  <option value="0">Option 1</option>
  <option value="1">Option 2</option>
</field>
```

Fill this fileld (chose few option) and save.

### Expected result

content history (preview) show changes/actual state 

### Actual result

content history doesnt show anything in colum with multiple values

### System information (as much as possible)

System operacyjny | Linux 8c69954b9ef8 5.9.0-0.bpo.2-amd64 #1 SMP Debian 5.9.6-1~bpo10+1 (2020-11-19) x86_64
Typ bazy danych | mysql
Wersja bazy danych | 5.5.5-10.4.12-MariaDB-1:10.4.12+maria~bionic
Metoda porównywania znaków w bazie danych | utf8mb4_general_ci
Metoda porównywania znaków przy połączeniu z bazą danych | utf8mb4_general_ci
Wersja PHP | 7.4.13
Serwer WWW | Apache/2.4.38 (Debian)
PHP działa na | apache2handler
Wersja Joomla! | Joomla! 3.9.23 Stable [ Amani ] 24-November-2020 15:00 GMT
Wersja Platformy Joomla! | Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
Klient użytkownika | Mozilla/5.0 (X11; Linux x86_64; rv:84.0) Gecko/20100101 Firefox/84.0

### Additional comments

I think the problem is in the method getFormValues
public/administrator/components/com_contenthistory/helpers/contenthistory.php:127
the method ignores that value can be an array.

**Comment by brianteeman** [comment]
There really isno point in keeping this issue open - the user is no longer on github and it was not a core issue

**Comment by brianteeman** [comment]
contenthistory does not consider changes in any custom fields

This is not a bug with field select multiple it is a missing feature for all custom fields.

@macjoom please change this from a bug to a feature request
